.mvn/extensions.xml

```diff
@@ -19,4 +19,14 @@
         <artifactId>quarkus-project-develocity-extension</artifactId>
         <version>1.1.9</version>
     </extension>
+    <extension>
+        <groupId>eu.maveniverse.maven.nisse</groupId>
+        <artifactId>extension3</artifactId>
+        <version>0.3.4</version>
+    </extension>
+    <extension>
+        <groupId>eu.maveniverse.maven.nisse</groupId>
+        <artifactId>extension4</artifactId>
```

Comment:
I suppose this is for Maven 4? It doesn't break anything to have it there now?

Reply:
No, the bean depends on an Maven 4 API interface which won't be available, the bean is silently ignored in Maven 3.